AmazonChimeSDKNoVideoCodecs.podspec

```diff
@@ -0,0 +1,14 @@
+Pod::Spec.new do |s|
+  s.name             = 'AmazonChimeSDKNoVideoCodecs'
+  s.version          = '0.25.1'
+  s.summary          = 'Amazon Chime SDK for iOS.'
```

Comment:
should we add no video codec specific to summary and description?
is the source link correct, seems it points to regular binary?

Reply:
> should we add no video codec specific to summary and description?

I think it will be nice to have specific summary, but when I check the media no vidoe codecs podspec file, we are using the same summary/description as the regular media binary, that's why I kept it the same for mobile.

> is the source link correct, seems it points to regular binary?

Yes, the mobile binary is the same one as regular one, the only difference is we are using no-video-codecs media as dependency in this podspec.

Reply:
I feel it's better to update for both.